src/test/java/io/cdap/delta/bigquery/BigQueryEventConsumerTest.java

```diff
@@ -482,6 +482,46 @@ public void testSoftDeletes() throws Exception {
     }
   }
 
+  @Test
+  public void testBigQuerySchemaNormalization() throws Exception {
+    String bucketName = "bqtest-" + UUID.randomUUID().toString();
```

Comment:
this is an existing issue, but it would be better if the unit test creates one bucket in a test wide setup method annotated with `@BeforeClass`, and deletes the bucket in an `@AfterClass` annotated method. The reduces the chance that tests fail due to a GCP transient error, but more importantly makes the tests cleaner, as we don't need to wrap everything in a try-finally

Reply:
this can be done in a separate PR though, as it's unrelated to column normalization and would require passing in a MockContext that has a different app name for each test to prevent conflicts.

Reply:
That's a good idea. Will address that in a separate PR